Mathlib/Tactic/Linter/UnusedTactic.lean

```diff
@@ -93,6 +93,7 @@ initialize allowedRef : IO.Ref (Std.HashSet SyntaxNodeKind) ←
     |>.insert `change?
     |>.insert `«tactic#adaptation_note_»
     |>.insert `tacticSleep_heartbeats_
+    |>.insert `Mathlib.Tactic.«tacticRename_bvar_→__»
```

Comment:
This is ok, especially since I do not think that `rename_bvar` is used much.  An alternative would be to add a bespoke check like the one for `swap_vars` that actually checks that `rename_bvar` is doing something.

Reply:
In fact, I think that `rename_bvar` does not check neither that the new name is different from the old one, nor that the old name actually is present.  These checks though should likely be performed by `rename_bvar`.

Reply:
After having thought more about this, I think that silencing the `unusedTactic` linter is correct.  The checks for `rename_bvar` (and `swap_vars`) should really be done by the respective tactics.

Thanks!

maintainer merge